### ISSUE TITLE
how to check league rank position

### ISSUE THREAD

**Issue by JackeyOL** [pseudonym117/Riot-Watcher#128]
As Riot deprecated the position api yesterday, how can I check the summoner's rank and position now by inputing summoner id?

**Comment by pseudonym117** [comment]
`watcher.league.by_id(encrypted_summoner_id)` - documented [here](https://riot-watcher.readthedocs.io/en/latest/riotwatcher/LeagueApiV4.html#riotwatcher._apis.LeagueApiV4.by_id) and [here](https://developer.riotgames.com/api-methods/#league-v4/GET_getLeagueEntriesForSummoner)

**Comment by haleemason** [comment]
The `by_id` endpoint is for pulling data by league_id, not encrypted_summoner_id right?

How do you pull data from the League endpoint by summoner currently? This is what I am trying: 
```
from riotwatcher import RiotWatcher, ApiError

api_key = 'RGAPI-key'
watcher = RiotWatcher(api_key)'
# Pull data from the LeagueApiV4
my_region = 'NA1'
encrypted_summoner_id = 'aiOiCVo3K_aErPYwx3ZjLXXmyMcrJL3whuHrY7dWqs_jTNc'

league_data = watcher.league.by_summoner(my_region, encrypted_summoner_id)
league_data
```

And I am getting an AttributeError:

```---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-25-9335977833b9> in <module>
      3 encrypted_summoner_id = 'aiOiCVo3K_aErPYwx3ZjLXXmyMcrJL3whuHrY7dWqs_jTNc'
      4 
----> 5 league_data = watcher.league.by_summoner(my_region, encrypted_summoner_id)
      6 league_data

AttributeError: 'LeagueApiV4' object has no attribute 'by_summoner'
# Check and filter to ranked matches only
```

However, I see a by_summoner function in [this file](https://github.com/pseudonym117/Riot-Watcher/blob/master/src/riotwatcher/_apis/LeagueApiV4.py) on line 68:

``` 
def by_summoner(self, region, encrypted_summoner_id):
        """
        Get league entries in all queues for a given summoner ID
        :param string region:                   the region to execute this request on
        :param string encrypted_summoner_id:    the summoner ID to query
        :returns: Set[LeagueEntryDTO]
        """
```

Also, when trying this same encryptedSummonerId on the [Riot API Endpoint](https://developer.riotgames.com/api-methods/#league-v4/GET_getLeagueEntriesForSummoner), it returns the following results:

RESPONSE BODY
```
[
    {
        "queueType": "RANKED_SOLO_5x5",
        "summonerName": "eydflt",
        "hotStreak": false,
        "wins": 17,
        "veteran": false,
        "losses": 23,
        "rank": "I",
        "tier": "PLATINUM",
        "inactive": false,
        "freshBlood": false,
        "leagueId": "e2064550-9e43-11e9-a30f-c81f66cf2333",
        "summonerId": "aiOiCVo3K_aErPYwx3ZjLXXmyMcrJL3whuHrY7dWqs_jTNc",
        "leaguePoints": 75
    }
]
```

**Comment by pseudonym117** [comment]
@haleemason double check that you have the latest version of `riotwatcher` installed via ` pip show riotwatcher`. Any version after `2.6` should have be correct, but `2.7.1` is the latest version.

If you are on the latest version and this is still an issue, please create a new issue.

**Comment by haleemason** [comment]
Hi, @pseudonym117. It appears that I was on an older version:
```
Name: riotwatcher
Version: 2.5.0
Summary: RiotWatcher is a thin wrapper on top of the Riot Games API for League of Legends.
Home-page: https://github.com/pseudonym117/Riot-Watcher
Author: AG Stephan
Author-email: None
License: MIT
Location: c:\users\new user\anaconda3\lib\site-packages
Requires: requests
```

After upgrading `Successfully installed riotwatcher-2.7.1` it now works as expected and I am able to pull the league data by summoner. Thanks so much for the support and for such a quick response!
